src/libstd/process.rs

```diff
@@ -253,6 +253,10 @@ impl Command {
     /// Builder methods are provided to change these defaults and
     /// otherwise configure the process.
     ///
+    /// The `PATH` from the Command's environment will be searched
+    /// if the program contains only a single path component, as for
+    /// `execvp`.
```

Comment:
I don’t get what “as for `execvp`“ part is supposed to mean. Why are you including references to POSIX specific functions in documentation of platform agnostic part of documentation?